openid-sse-framework-1_0.txt

```diff
@@ -557,39 +552,44 @@ Tulshibagwale, et al.   Expires December 10, 2021               [Page 9]
 
 
 
-Tulshibagwale, et al.   Expires December 10, 2021              [Page 10]
+
+
+
+
+
+Tulshibagwale, et al.        Standards Track                   [Page 10]
 
                           openid-sse-framework                 June 2021
 
 
    Transmitters supporting Discovery MUST make a JSON document available
-   at the path formed by inserting the string "/.well-known/sse-
-   configuration" into the Issuer between the host component and the
-   path component, if any.  The syntax and semantics of ".well-known"
-   are defined in [RFC5785].  "sse-configuration" MUST point to a JSON
+   at the path formed by inserting the string /.well-known/sse-
+   configuration into the Issuer between the host component and the path
```

Comment:
I think the quotes are important here, otherwise the spec cannot be read unambiguously